setup/mail-postfix.sh

```diff
@@ -210,6 +210,16 @@ tools/editconf.py /etc/postfix/main.cf \
 tools/editconf.py /etc/default/postgrey \
 	POSTGREY_OPTS=\"'--inet=127.0.0.1:10023 --delay=180'\"
 
+
+# We are going to setup a newer whitelist for postgrey, the version included in the distribution is old
+cat > /etc/cron.monthly/mailinabox-postgrey-whitelist << EOF;
+#!/bin/bash
+# Mail-in-a-Box
+curl http://postgrey.schweikert.ch/pub/postgrey_whitelist_clients --output /etc/postgrey/whitelist_clients -ss
```

Comment:
You can also use the https link here: https://postgrey.schweikert.ch/pub/postgrey_whitelist_clients

Reply:
Yes good points. 

I did -ss from memory and it works, but proper usage is -sS I’ll correct both issues

Reply:
My only other concern would be that miab usually uses "hide_output" when installing parts.  If -sS is used does that produce output if there was a failure?

Reply:
We can’t use the hideoutput here unless we reference the scripts from the repo. We don’t know the path at run time and the user might remove the repo.

Reply:
Of course, this is running as cron.monthly.

But is there the possibility that line 221, where we run the mailinabox-postgrey-whitelist for the first time, will cause some output?

Reply:
> But is there the possibility that line 221, where we run the mailinabox-postgrey-whitelist script for the first time, will cause some output?
It wouldn't post any output.
>I'm also thinking about what would happen should this fail, will the user be left without a whitelist for a month until the next attempt to run?
I did some testing with this. By default when the script receives a 404 it won't produce any output, it will just dump the error page to the output file. Adding the --fail option changes that behavior, however this does produce output on the console.
To fix that I've redirect both stderr and out to null.